scene/main/viewport.cpp

```diff
@@ -3685,6 +3686,13 @@ bool SubViewport::is_using_xr() {
 
 void SubViewport::set_size(const Size2i &p_size) {
 	_set_size(p_size, _get_size_2d_override(), Rect2i(), _stretch_transform(), true);
+
+	if (get_parent()) {
+		SubViewportContainer *c = Object::cast_to<SubViewportContainer>(get_parent());
+		if (c) {
+			c->minimum_size_changed();
+		}
+	}
```

Comment:
```suggestion
	SubViewportContainer *c = Object::cast_to<SubViewportContainer>(get_parent());
	if (c) {
		c->minimum_size_changed();
	}
```
This should still work even if parent is null, AFAIK.